src/tools/r2rdump/CoreDisTools.cs

```diff
@@ -103,6 +116,14 @@ public void Dispose()
             }
         }
 
+        /// <summary>
+        /// Parse a single instruction and return the RVA of the next instruction.
+        /// </summary>
+        /// <param name="rtf">Runtime function to parse</param>
+        /// <param name="imageOffset">Offset within the PE image byte array</param>
+        /// <param name="rtfOffset">Instruction offset within the runtime function</param>
+        /// <param name="instruction">Output text representation of the instruction</param>
+        /// <returns></returns>
```

Comment:
Can you document here that the instruction size is returned?

Reply:
Good suggestion, fixed in 4th commit.